### PR TITLE
Doc reorganization

### PR DESCRIPTION
**Pull request by randyh62**
reorganized and created RST doc contents.

### CODE REVIEW

**Review by kiritigowda — APPROVED** [review]

**Comment by lpaoletti** [comment]
This looks good. 
Did notice that most of the TOC titles don't line up with the actual topic titles. 
Especially in here:
image

File: docs/index.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  :description: MIVisionX API
+  :keywords: MIVisionX, ROCm, API, reference, data type, support
+
+.. _index:
+
+******************************************
+MIVisionX documentation
+******************************************
+
+AMD MIVisionX toolkit is a comprehensive set of computer vision and machine intelligence libraries, utilities, and applications bundled into a single toolkit. MIVisionX delivers AMD's implementation of `Khronos OpenVX <https://www.khronos.org/openvx/>`_, and a Convolutional Neural Net model compiler & optimizer supporting `ONNX <https://onnx.ai/>`_ and `Khronos NNEF <https://www.khronos.org/nnef>`_ exchange formats. The MIVisionX toolkit allows for rapid prototyping and deployment of optimized computer vision and machine learning inference workloads on a wide range of computer hardware, including small embedded x86 CPUs, APUs, discrete GPUs, and heterogeneous servers. For more information refer to :ref:`what-is-mivisionx`.
```
Comment (lpaoletti) [inline docs/index.rst:11]: Add comma:
 For more information, refer to

File: docs/install/amd_openvx-install.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Build Instructions
+==================
+
+Build this project to generate AMD OpenVX library 
+
+* Refer to `openvx/include/VX <https://github.com/ROCm/MIVisionX/tree/master/amd_openvx/openvx/include>`_ for Khronos OpenVX standard header files.
+* Refer to `openvx/include/vx_ext_amd.h <https://github.com/ROCm/MIVisionX/tree/master/amd_openvx/openvx/include/vx_ext_amd.h>`_ for vendor extensions in AMD OpenVX library
+
+Build using `Visual Studio`
```
Comment (lpaoletti) [inline docs/install/amd_openvx-install.rst:29]: Microsoft Visual Studio with no italics

File: docs/install/model-compiler-install.rst [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+.. _model-compiler-install:
+
+*********************************************
+Model compiler installation and configuration
+*********************************************
+
+The following describes installing and setting up the model compiler & optimizer and neural net models for Linux. 
+
+Pre-requisites
```
Comment (lpaoletti) [inline docs/install/model-compiler-install.rst:13]: no hyphen

**Pull request merged**